lib/qri/qri.go

```diff
@@ -14,24 +16,26 @@ import (
 const ModuleName = "qri.sky"
 
 // NewModule creates a new qri module instance
-func NewModule(ds *dataset.Dataset, secrets map[string]interface{}) *Module {
-	return &Module{ds: ds, secrets: secrets}
+func NewModule(ds *dataset.Dataset, secrets map[string]interface{}, infile cafs.File) *Module {
```

Comment:
I'll have to clean this up in relation to #2, but the addition of `infile` is obviously needed